### PR TITLE
Release notes for 0.15.1

### PR DESCRIPTION
**Pull request by yboaron**

### CODE REVIEW

File: src/content/community/releases/_index.en.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.15.1
+
+This is a bugfix release:
+
+* Fixed IPsec pluto crash when remote endpoint is unstable.
```
Comment (tpantelis) [inline src/content/community/releases/_index.en.md]: To be consistent with https://github.com/submariner-io/submariner-website/pull/944/files.

```suggestion
* Fixed issue which caused the IPsec pluto process to crash when the remote endpoint was unstable.
```

File: src/content/community/releases/_index.en.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ weight = 40
...
+* Avoid cableEngine cleanup when the gateway pod is deleted.
```
Comment (tpantelis) [inline src/content/community/releases/_index.en.md]: To be consistent with https://github.com/submariner-io/submariner-website/pull/944/files - I think you're referring to the same issue.

```suggestion
* Fixed issue where a Gateway pod restart caused data path disruption.
```

File: src/content/community/releases/_index.en.md [hunk 3]
```diff
@@ -5,6 +5,13 @@ weight = 40
...
+
```
Comment (tpantelis) [inline src/content/community/releases/_index.en.md:16]:
```suggestion
Service Discovery now publishes DNS records for pods that are not ready for headless services based on the setting of the `publishNotReadyAddresses` flag on the Service.
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: src/content/community/releases/_index.en.md (hunk 2 above)
Comment (yboaron) [reply src/content/community/releases/_index.en.md]: I think it should be gateway deleted and not gateway restarted.

cc @sridhargaddam

Comment (tpantelis) [reply src/content/community/releases/_index.en.md]: They're essentially equivalent b/c a deletion causes a restart. But a restart can occur for other reasons so I think it's more accurate here.

**Review by skitt — APPROVED** [review]

**Review by sridhargaddam — APPROVED** [review]

File: src/content/community/releases/_index.en.md [hunk 4]
```diff
@@ -5,6 +5,15 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.15.1
+
+This is a bugfix release:
+
+* Fixed issue which caused the IPsec pluto process to crash when the remote endpoint was unstable.
+* Fixed issue where a Gateway pod deletion caused data path disruption..
```
Comment (tpantelis) [inline src/content/community/releases/_index.en.md]: Duplicate period at the end. 

Not trying to nitpick but I still think "restarted" is more appropriate. We delete pods in testing to effect a restart but I don't think an end user would do that. From the PR https://github.com/submariner-io/submariner/pull/2499, the scenario was due to a gateway failover restart, not explicit deletion of a pod.

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: src/content/community/releases/_index.en.md (hunk 4 above)
Comment (yboaron) [reply src/content/community/releases/_index.en.md]: > Duplicate period at the end.

Ooops, thanks.

 https://github.com/submariner-io/submariner/pull/2499 avoid DP cleanup when GW is terminated due to SIGINT or SIGERM signals. 

'Kubectl delete pod'  command triggers a SIGTERM signal to pod's main process ID.

So, ^^ PR should help when pod is deleted and also in some GW restart use cases. 

I'll update this item.

**Review by tpantelis — APPROVED** [review]

**Pull request merged**